index.html

```diff
@@ -100,196 +107,329 @@ <h2 id="connect">Let's Connect</h2>
-          />
-          <h2>Multi-Post Stories</h2>
-          <ul class="experience">
-            <li>Canopy</li>
-            <li>&#x2022;</li>
-            <li>Back End Dev</li>
-            <li>&#x2022;</li>
-            <li>2015</li>
-          </ul>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="technologies">
-            <li>html</li>
-            <li>css</li>
-            <li>javascript</li>
+          <ul class="work-card__ul" id="work__second">
+            <li class="work-card__header">
+              <picture>
+                <source
+                  srcset="images/work_2_lg.png"
+                  media="(min-width: 768px)"
+                />
+                <img
+                  src="images/work_2.jpg"
+                  alt="work_multi-post_stories_daily_personalized_reads"
+                />
+              </picture>
+            </li>
+            <li class="work-card__body">
+              <ul>
+                <li class="work-card__title">Multi-Post Stories</li>
+                <li>
+                  <ul class="work-card__experience mobile">
+                    <li>Canopy</li>
+                    <li>&#x2022;</li>
+                    <li>Back End Dev</li>
+                    <li>&#x2022;</li>
+                    <li>2015</li>
+                  </ul>
+                  <ul class="work-card__experience desktop">
+                    <li>FACEBOOK</li>
+                    <li>&#x2022;</li>
+                    <li>Full Stack Dev</li>
+                    <li>&#x2022;</li>
+                    <li>2015</li>
+                  </ul>
+                </li>
+                <li class="work-card__description mobile">
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required.
+                </li>
+                <li class="work-card__description desktop">
+                  Experimental content creation feature that allows users to add
+                  to an existing story over the course of a day without spamming
+                  their friends.
+                </li>
+                <li>
+                  <ul class="work-card__technologies mobile">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                  </ul>
+                  <ul class="work-card__technologies desktop">
+                    <li>html</li>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                  </ul>
+                </li>
+                <li class="btn-container">
+                  <button type="button" class="btn">
+                    <a href="#">See Project</a>
+                  </button>
+                </li>
+              </ul>
+            </li>
```

Comment:
- [ ] Kindly for semantic tags, instead of using `ul` for wrapping your items inside use `section` or `article` depending on the information that you are using inside. For example, `ul` tags are for listing items, like following a recipe. But in this list of cards, you have articles for example. There's no list of items, so `ul` and `li` are not semantic in this section. Kindly replace it with the corresponding one. [section ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)and [article ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article)